models.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional, TypedDict
+from dane.provenance import Provenance
+
+
+# returned by callback()
+class CallbackResponse(TypedDict):
+    state: int
+    message: str
+
+
+# These are the types of output this worker (possibly) provides (depending on configuration)
+class OutputType(Enum):
+    # name of output type, should just have a significant name, no other restrictions
+    # (as far as I understand)
```

Comment:
Correct. The idea is that mainingful bits of output will be in separate folders, inside the resulting tar archive